versionedController/v1/email/email.go

```diff
@@ -54,7 +54,7 @@ func (emailController *EmailController) Get(c *gin.Context) {
 			"error": err.Error(),
 		})
 		// Redirecting user back to UI if access token is not valid
-		c.Redirect(http.StatusUnauthorized, redirectURL)
+		c.Redirect(http.StatusPermanentRedirect, redirectURL)
```

Comment:
Why change the error code?

Reply:
Same comment as https://github.com/mayadata-io/kubera-auth/pull/45/files#r592392465